example/api_context_save_example.php

```diff
@@ -15,5 +15,6 @@
 const DEVICE_DESCRIPTION = 'Server 1';
 const PERMITTED_IPS = [];
 
+date_default_timezone_set('UTC'); // When using php5.6
```

Comment:
hmm, then we probably want to set this as a default for all times the date is used... In some entry point of the SDK perhaps, or just create a date wrapper to use there %)